src/server/index.ts

```diff
@@ -12,6 +16,20 @@ const main = async () => {
     console.log(`Listening on port ${app.get('port')} 🚀`);
     console.log('  Press Control-C to stop\n');
   });
+
+  app.use(express.json());
+  app.use(cookieParser());
+  app.use(express.urlencoded({ extended: true }));
+  app.use(passport.initialize());
+  app.use(userRouter)
+
+  const port = process.env.PORT || 8000;
+  app.get('/', (req, res) => {
+    res.json({
+        message: "Node Cookie JWT Service"
+    })
+})
+
```

Comment:
This should be added to config/createServerFile

Reply:
resolved!